.travis.yml

```diff
@@ -12,48 +12,73 @@ branches:
     # This is enabled to build pull requests.
     - master
 
+env:
+  - TEST_KIND="mjtest-lexer"
```

Comment:
Why do we need this? Everywhere in the matrix the `TEST_KIND` is defined. This shouldn't be required.